guides/common/modules/ref_prerequisites-for-using-ipxe.adoc

```diff
@@ -0,0 +1,30 @@
+[id="prerequisites-for-using-ipxe_{context}"]
+= Prerequisites for using iPXE
+
+You can use iPXE to boot virtual machines on hypervisors that use iPXE as primary firmware.
+Furthermore, you can configure PXELinux to chainboot iPXE in BIOS mode and boot by using the HTTP protocol if you have the following restrictions that prevent you from using PXE:
```

Comment:
I'm not sure "Furthermore" is the right word here because it implies a relationship between the sentences that does not exist.

Suggestion: "You can chainboot iPXE in BIOS mode and then boot by using HTTP if the following restrictions prevent you from using PXE:"

Reply:
I'll rewrite the intro to make clearer what I'm trying to say.

Reply:
@apinnick I've moved the restrictions to the introductory concept module and kept two use cases in the prerequisites. What do you think? Is it clearer?